presto-main/src/main/java/com/facebook/presto/sql/rewrite/ShowQueriesRewrite.java

```diff
@@ -508,12 +510,14 @@ protected Node visitShowCreate(ShowCreate node, Void context)
                 ConnectorTableMetadata connectorTableMetadata = metadata.getTableMetadata(session, tableHandle.get()).getMetadata();
 
                 Map<String, PropertyMetadata<?>> allColumnProperties = metadata.getColumnPropertyManager().getAllProperties().get(tableHandle.get().getConnectorId());
-
+                Set<String> reservedIdentifiers = ReservedIdentifiers.reservedIdentifiers();
                 List<TableElement> columns = connectorTableMetadata.getColumns().stream()
                         .filter(column -> !column.isHidden())
                         .map(column -> {
                             List<Property> propertyNodes = buildProperties(objectName, Optional.of(column.getName()), INVALID_COLUMN_PROPERTY, column.getProperties(), allColumnProperties);
-                            return new ColumnDefinition(new Identifier(column.getName()), column.getType().getDisplayName(), column.isNullable(), propertyNodes, Optional.ofNullable(column.getComment()));
+                            Identifier identifier = reservedIdentifiers.contains(column.getName().toUpperCase(ENGLISH)) ? QueryUtil.quotedIdentifier(column.getName()) :
```

Comment:
I think it's ok to just create quotedIdentifier for all column names which would avoid the check and avoid keeping a set as well. MySQL also quote all column names in show create (using backquote).

Reply:
It makes a lot of sense and we can avoid maintaining the set. I made changes accordingly. If presto supports case sensitive identifier in the future, we can address the show create functionality accordingly.

Reply:
@abhiseksaikia Please, update the PR title and description as well as commit message to reflect the change.

Reply:
@mbasmanova  I made the changes.